duouniversal/client.go

```diff
@@ -218,28 +218,60 @@ func newStrictTLSTransport() *http.Transport {
 	}
 }
 
-func NewClient(clientId, clientSecret, apiHost, redirectUri string) (*Client, error) {
-	return NewClientDuoCodeAttribute(clientId, clientSecret, apiHost, redirectUri, true)
+func NewClient(clientId, clientSecret, apiHost, redirectUri string, opts ...clientOptions) (*Client, error) {
```

Comment:
Hi Aaron, this will no affect backwards compatibility and the options params can be excluded. Note the usage of `newClient` on line 231. I aimed to preserve backwards compatibility in this change